### ISSUE TITLE
Maintainer?

### ISSUE THREAD

**Issue by excid3** [ctran/annotate_models#913]
@ctran would you like some help maintaining the gem and releasing updates? 

I use annotate in almost every app (thank you!) and maintain ActsAsTenant and several other gems. Just want to help keep things running smoothly!

**Comment by excid3** [comment]
Bump @ctran

**Comment by KieranP** [comment]
Bump @ctran  - this gem is preventing me from upgrading to Rails 7

**Comment by FlorianBruniaux** [comment]
@ctran Yes +1. My team and I (several ror developers) would love to help maintain the gem with you!
@ocarta-l , @clement1234, @winwave, @floriansr, @yao0204 💪🏻

**Comment by jeromepl** [comment]
@ctran it's been a year since the last comment here, any update? This gem is still actively used by many and could benefit from having more maintainers 🙂

**Comment by ChristianClassen** [comment]
@ctran Do you still maintain this gem?

**Comment by drwl** [comment]
Hopefully this is not a bad practice, but as a semi-maintainer of this gem I've spent some time forking this and making big improvements on the gem here https://github.com/drwl/annotaterb.

**Comment by justinko** [comment]
@drwl Thank you! I'm gonna try it out.

**Comment by ctran** [comment]
@drwl Why are these changes submitted merged back?

**Comment by drwl** [comment]
> @drwl Why are these changes submitted merged back?

What do you mean by this?

**Comment by miguelperez** [comment]
> > @drwl Why are these changes submitted merged back?
> 
> What do you mean by this?

I am sorry to interject here, But I think he meant something like why haven't those changes been merged back into annotate. Instead of keeping them in a fork,

**Comment by drwl** [comment]
@miguelperez no problem. I wanted to make significant breaking changes so I did on my own repo and fork.
